src/Assets/LightSkyDefense/Scripts/TowerAI.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Assets
+{
+    public class TowerAI : MonoBehaviour
+    {
+        public Rigidbody Projectile;
+        public Transform ProjectileSpawn;
+        public Transform Target = null;
+        private IEnumerator _coroutine;
+        public float ReloadTime = 2;
+        public float BulletSpeed = 5;
+
+        void OnTriggerEnter(Collider target)
+        {
+            Enemy enemyScript = target.gameObject.GetComponent<Enemy>();
+            if (enemyScript == null) return;
+            Target = target.transform;
+        }
+
+        void OnTriggerExit(Collider target)
+        {
+            Enemy enemyScript = target.gameObject.GetComponent<Enemy>();
+            if (enemyScript == null) return;
```

Comment:
Can we add newlines around if statements please.

```suggestion

            if (enemyScript == null) return;

```